AcadeMeData/migrations/0004_messages_test_data.py

```diff
@@ -9,7 +9,7 @@ class Migration(migrations.Migration):
 
     def generate_msgdata(apps, schema_editor):
         from AcadeMeData.models import Messages, User
-        userrnd1 = User.create_user('name', 'email', 'pass', 's', 'kaki', 'cs')
+        userrnd1 = User.create_user('name', 'email', 'pass', 'kaki', 'cs')
         test_data = [
             (1, userrnd1, 'random message'),
         ]
```

Comment:
This file has been modified to be compatible with the latest User model
see that the field Type is no longer in the user model so I have deleted it from the test data files 
('S')